docs/proposed/psa-conditional-inclusion-c.md

```diff
@@ -195,7 +201,7 @@ The boolean symbol mechanism proposed here can be translated to a list of JSON c
 
 #### Naming of symbols
 
-The names of [elliptic curve symbols](#configuration-symbols-for-curves) are a bit weird: `SECP_R1_256` instead of `SECP256R1`. Should we make them more classical, but less systematic?
+The names of [elliptic curve symbols](#configuration-symbols-for-elliptic-curves) are a bit weird: `SECP_R1_256` instead of `SECP256R1`, `MONTGOMERY_255` instead of `CURVE25519`. Should we make them more classical, but less systematic?
```

Comment:
SECP_R1_256 at least has the same symbols in it as SECP256R1. It is a little weird, but at least understandable. I've never seen CURVE25519 called Mongomery 255, and that one makes more sense to change the name. DJB does now recommend the name Curve25519 for the curve itself, and X25519 for the DH function (this was not what the original paper said).